gst/nnstreamer/nnstreamer_plugin_api_impl.c

```diff
@@ -1784,3 +1784,63 @@ gst_buffer_n_tensor (GstBuffer * buffer)
 
   return num_mems;
 }
+
+/**
+ * @brief Parses a configuration file and sets the corresponding properties on a GObject.
+ *
+ * This function reads the contents of the configuration file located at the given path
+ * and sets the properties of the specified GObject based on the configuration data.
+ *
+ * @param config_path The path to the configuration file.
+ * @param object      The GObject on which to set the properties.
+ *
+ * @note The responsibility of managing the memory of the GObject passed as a parameter
+ *       lies outside this function.
+ */
+
+void
+gst_tensor_parse_config_file (const gchar * config_path, const GObject * object)
+{
+  GError *error = NULL;
+  gchar *config_data = NULL;
+  gchar **lines = NULL;
+  gchar **line = NULL;
+  GObjectClass *g_object_class = G_OBJECT_GET_CLASS (object);
+
+  if (!g_file_get_contents (config_path, &config_data, NULL, &error)) {
+    GST_DEBUG ("Failed to read config file: %s\n", error->message);
+    g_error_free (error);
+    return;
+  }
+
+  lines = g_strsplit (config_data, "\n", -1);
+  g_free (config_data);
+
+  /** Iterate over each line */
+  for (line = lines; *line; ++line) {
+    gchar **parts = g_strsplit (*line, "=", 2);
+    if (g_strv_length (parts) == 2) {
+      gchar *property_name = g_strstrip (g_strdup (parts[0]));
+      gchar *property_value = g_strstrip (g_strdup (parts[1]));
+
+      GParamSpec *pdata =
+          g_object_class_find_property (g_object_class, property_name);
+      if (pdata != NULL) {
+        GValue prop_value = G_VALUE_INIT;
+        g_value_init (&prop_value, G_TYPE_STRING);
+        g_value_set_string (&prop_value, property_value);
+        g_object_set_property (G_OBJECT (object), pdata->name, &prop_value);
```

Comment:
I like this changes. Good!
We will extend this method to various elements such as tensor_filter, tensor_if, and tensor_transform.
The property type may vary not only string but also boolean or int and etc.
So, let's set the property according to type of GParamSpec.

Reply:
I shall include some common types: int, unit, float, double, boolean, string